### DISCUSSION TITLE
Can't access fresh Digital Ocean one-click droplet

### DISCUSSION THREAD

**Discussion by davidbwaters** [directus/directus#6398] [Archive]
Hi, I can't access fresh Digital Ocean one-click droplet from the public ur.  I have tried restarting from PM2 and upgrading from NPM.  The database tables are installed.  I'm not sure what the issue is.

**Comment by rijkvanzanten** [comment]
Hi @davidbwaters! It's a little hard to help troubleshoot this without knowing a little more about the troubles 🤔 

What are you seeing when you try opening the public URL? Are there any errors in the devtools, or any other errors shown somewhere?

**Reply by davidbwaters** [reply]
I fixed it.  It wasn't a Directus issue.  IDK how to close a discussion. It was a DNS nxdomain issue in Chrome/Firefox that I fixed by pointing a subdomain and getting HTTPS working, if anyone else experiences.  Thanks.
